CHANGELOG.md

```diff
@@ -5,6 +5,30 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## dev
```

Comment:
release?

Reply:
mentioned on slack: separate PR in comping to speed up reviews, while I pick a new release name

Reply:
here: https://github.com/nf-core/sarek/pull/1245 just looking through the maps now